src/Connector.sol

```diff
@@ -141,7 +141,7 @@ contract CentrifugeConnector {
         uint64 poolId,
         bytes16 trancheId,
         address user,
-        uint256 validUntil
+        uint64 validUntil
```

Comment:
This was wrong and often `validUntil` was named `amount`, probably from a copy-paste from `transfer` I guess

Reply:
Good find!